### PR TITLE
JUBE: add v2.6.x

### PR DESCRIPTION
**Pull request by fspiga**
Adding JUBE 2.6.x

ChangeLogs:
- 2.6.0: https://apps.fz-juelich.de/jsc/jube/jube2/docu/release_notes.html#version-2-6-0
- 2.6.1: https://apps.fz-juelich.de/jsc/jube/jube2/docu/release_notes.html#version-2-6-1

### CODE REVIEW

**Review by tldahlgren — CHANGES_REQUESTED** [review]
I am able to confirm version 2.6.0 sha256 but get a different value from `spack checksum` for version 2.6.1 (348ebeb4db0ea56889a5d13bf49e394fccc0992f620e8c0c61c40185db13ad64).

**Review by adamjstewart — DISMISSED** [review]
Other than the checksum, the rest looks good

File: var/spack/repos/builtin/packages/jube/package.py [hunk 1]
```diff
@@ -14,6 +14,16 @@ class Jube(PythonPackage):
     homepage = "https://www.fz-juelich.de/jsc/jube/"
     url = "https://apps.fz-juelich.de/jsc/jube/jube2/download.php?version=2.2.2"
 
+    version(
+        "2.6.1",
+        sha256="32846616b01450e7ab678a1b3a7b34bcbb2311591a095d2b27e6d3a39110c07c",
```
Comment (RMeli) [inline var/spack/repos/builtin/packages/jube/package.py]:
```suggestion
        sha256="348ebeb4db0ea56889a5d13bf49e394fccc0992f620e8c0c61c40185db13ad64",
```

**Review by tldahlgren — APPROVED** [review]
Can now confirm both sha256s after adding the suggested fix through the web interface.

**Pull request merged**